app/src/main/java/br/ufg/inf/pes/healthhelp/view/HomeActivity.java

```diff
@@ -0,0 +1,64 @@
+package br.ufg.inf.pes.healthhelp.view;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Toast;
+
+import br.ufg.inf.pes.healthhelp.service.MensagemNaoImplementado;
+import br.ufg.pes.healthhelp.R;
+
+
+public class HomeActivity extends AppCompatActivity {
+
+    private RecyclerView mRecyclerView;
+    private RecyclerView.Adapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.home_activity);
+
+    }
+
+    public void visitasMedicas(View view){
+
+        Context context = getApplicationContext();
+        CharSequence text = "Visualizar visitas medicas nao implementado";
+        MensagemNaoImplementado.MostraMensagemNaoImplementado(context, text);
```

Comment:
Utilize a própria instância da activity com o `this` como contexto. Coloque a mensagem diretamente como parâmetro. Isso vai evitar criar 2 variáveis extras apenas pra exibir um Toast.